DotNetCoreMasters/DotNetCoreMasters/Controllers/ItemsController.cs

```diff
@@ -8,16 +11,62 @@
 
 namespace DotNetCoreMasters.Controllers
 {
-    [ApiController]
     [Route("[controller]")]
-    public class ItemsController : Controller
+    public class ItemsController : ControllerBase
     {
-        public IActionResult GetAll(int userId)
+        [HttpGet]
+        public IActionResult GetAll()
         {
-            var res = new ItemService();
-            var listOfStrings = res.GetAll(userId);
+            var listItems = new List<string>()
+            {
+                "Item1",
+                "Item2",
+                "Item3"
+            };
 
-            return Ok(listOfStrings);
+            return Ok(listItems);
         }
+
+        [HttpGet("/items/{itemId}")]
```

Comment:
because of [Route("[controller]")] on the controller level, there is no need for "/items/". Please clean up the routes.